docs/user-documentation/moderne-platform/how-to-guides/reporting.md

```diff
@@ -0,0 +1,61 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Reporting
+
+## Overview
+
+Moderne offers some reports:
+
+* **Usage by organization** - every recipe run and associated commit by every user for a specific organization. 
+
+## Usage by organization
+The generated report includes every recipe run and associated commit for a given organization. This report includes
+the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. 
+If any commit was created in association with the recipe run details of the commit are included: the type of 
+commit, status of the commit job, as well as other statistics of the commit job.
+
+The usage report is accessible from `https://<TENANT>.moderne.io/devcenter/{organization}`
```

Comment:
This is how the image will be diplayed 
<img width="983" alt="Screenshot 2024-12-10 at 11 47 39 AM" src="https://github.com/user-attachments/assets/8e47ed0a-e24e-4b5d-af00-71a22886ce14">

Reply:
Updated screenshot
<img width="1022" alt="Screenshot 2024-12-11 at 8 40 50 AM" src="https://github.com/user-attachments/assets/fa039a8a-7ea9-4b27-8901-e14c09ab6722" />